definitions/linux.yaml

```diff
@@ -128,8 +128,8 @@ sources:
 - type: ARTIFACT
   attributes:
     names:
-      - UsersShellConfigs
-      - RootUserShellConfigs
+      - UsersShellHistory
+      - RootUserShellHistory
 labels: [Configuration Files]
```

Comment:
It's consistent with the UsersShellHistory and RootUserShellHistory artifact labels. Happy to change them, but I'm not sure what category you'd prefer.